gitops/overlays/olivier/configs/frontend-config.conf

```diff
@@ -0,0 +1,30 @@
+LOG_LEVEL=debug
+
+#
+# Global mock configuration
+#
+ENABLED_MOCKS=cct,lookup,power-platform,raoidc,wsaddress
+
+#
+# Session/redis configuration
+#
+SESSION_STORAGE_TYPE=redis
+REDIS_URL=redis://redis-dev
+SESSION_TIMEOUT_SECONDS=60
+SESSION_TIMEOUT_PROMPT_SECONDS=30
```

Comment:
For this pseudo-environment, I think we should use file-based sessions. There's no need to deploy a full-blown redis server for what is effectively a prototype.

Reply:
Done